js/client-chat.js

```diff
@@ -148,6 +148,11 @@
 					e.preventDefault();
 					e.stopPropagation();
 				}
+			} else if (app.user.lastPM && (textbox.value === '/reply' || textbox.value === '/r') && e.keyCode === 32) { // '/reply ' is being written
+				e.preventDefault();
+				e.stopPropagation();
+				var val = textbox.value = '/pm ' + app.user.lastPM + ', ';
```

Comment:
```
var val = '/pm ' + app.user.lastPM + ', ';
textbox.value = val;
```

Don't use fancy tricks on client; you don't know what could cause a deoptimization :|